src/RemoteTech/UI/AbstractWindow.cs

```diff
@@ -187,6 +188,16 @@ public virtual void Draw()
                     onPositionChanged.Invoke();
                     backupPosition = Position;
                 }
+
+                // Set ship control lock if one rt input is in focus
+                if (GUI.GetNameOfFocusedControl().StartsWith("rt_"))
```

Comment:
strings that are well known like this could be pulled out into a constant, mainly so we can name it